Cargo.toml

```
@@ -19,4 +19,6 @@ clap = "2.33.0"
 crossterm = "0.17.3"
 tui = {  version = "0.9.1", default-features = false, features = ["crossterm"] }
 # Flexible error handling
-anyhow = "1.0.31"
\ No newline at end of file
+anyhow = "1.0.31"
+# Watch files and auto-reload on changes
+inotify = "0.8.3" # Linux
```

```suggestion

[target.'cfg(target_os = linux)'.dependencies]
inotify = "0.8.3"
```

This might work for platform-dependent dependencies. But it isn't tested, and tbh, is useless as of now. But it would be nice to have something like this in the future.

---

Applied.